CHANGELOG.md

```diff
@@ -1,8 +1,28 @@
 # Changelog
 All changes to the project will be documented in this file.
 
-## [4.0.x] - Not Yet Released
+## [4.0.130103] - 2020-06-01
+[View Complete Diff of Changes](https://www.github.com/dotnet/roslyn/compare/3f2b20c65d32a59ca6bbc68b788a31ed38576d8e...61e5536d636f5b13c74f710d658bb320b224026d)
```

Comment:
when I go to this link I see "There isn’t anything to compare."

Reply:
haha good catch. I had my prfinder set to Roslyn =p

Reply:
I should really update it to autodiscover from git remotes...